artifactory/commands/setup/setup.go

```diff
@@ -237,7 +237,7 @@ func (sc *SetupCommand) configurePoetry() error {
 //
 // Note: Custom configuration file can be set by setting the NPM_CONFIG_USERCONFIG environment variable.
 func (sc *SetupCommand) configureNpmPnpm() error {
-	repoUrl := commandsutils.GetNpmRepositoryUrl(sc.repoName, sc.serverDetails.ArtifactoryUrl)
+	repoUrl := commandsutils.GetNpmRepositoryUrl(sc.repoName, sc.serverDetails.ArtifactoryUrl) + "/"
```

Comment:
If the scenario where the ArtifactoryUrl already contains a trailing slash is possible, then lets address it.